ratatui-widgets/src/chart.rs

```diff
@@ -170,6 +170,85 @@ pub enum GraphType {
     Bar,
 }
 
+/// Used to specify a range of values that will be drawn in another color.
+/// If the value does not fall inside any range specified the color will fallback to the style
+/// foreground color.
+///
+/// Has no effect in any [`GraphType`] other than [line](GraphType::Line)
+#[derive(Debug, Default, Clone, PartialEq)]
+pub struct MultiColorLineConfig {
```

Comment:
I'd like to rename this to `MultiColorLine` since we don't usually use `Config` suffix with types.

```suggestion
pub struct MultiColorLine {
```